### PR TITLE
Add method getExampleInput() to Format class

### PR DESCRIPTION
**Pull request by koppor**
I added `getExampleInput()` to the Formatter class to improve #1050.

### CODE REVIEW

File: src/test/java/net/sf/jabref/logic/formatter/bibtexfields/UnicodeConverterTest.java [hunk 1]
```diff
@@ -27,6 +27,12 @@ public void testBasic() {
     }
 
     @Test
+    public void testEmpty() {
```
Comment (tobiasdiez) [inline src/test/java/net/sf/jabref/logic/formatter/bibtexfields/UnicodeConverterTest.java]: not formatExample? The empty string should be covered by the general tests.

**Comment by tobiasdiez** [comment]
LGTM :+1:
Could you please also add a test in FormatterTests that  getExampleInput does not return null or an empty string?
